.github/actions/bump-version/action.js

```diff
@@ -6,10 +6,10 @@ function bumpVersion(currentVersion, bumpType, prerelease) {
   if (prerelease) {
     newVersion = `${newVersion}.${prerelease}`;
   }
-  core.setOutput('previous_version', currentVersion)
-  core.setOutput('previous_version_tag', `v${currentVersion}`)
+  core.setOutput('previous_version', currentVersion);
```

Comment:
Made changes in here initially and backed them out, but prettier had some updates.